pom.xml

```diff
@@ -129,6 +129,10 @@
               <mainClass>io.anserini.search.SimpleTweetSearcher</mainClass>
               <id>SimpleTweetSearcher</id>
             </program>
+            <program>
+              <mainClass>io.anserini.search.SimpleImpactSearcher</mainClass>
```

Comment:
I think we should remove this - see more below...

Reply:
seems if we remove this, it will not be included in fatjar?

Reply:
seems if we remove this, it will not included in fatjar?

Reply:
Really? I guess I might have forgotten this before? Where is this documented?